examples/nested_inline_tasksets.py

```diff
@@ -23,3 +23,4 @@ def stop(self):
         def stats(self):
             self.client.get("/stats/requests")
 
+    tasks = [TopLevelTaskSet]
```

Comment:
We could also decorate the `TopLevelTaskSet` with `@task` instead of setting the `tasks` attribute.